src/plugins/condition/utils/operations.js

```diff
@@ -298,12 +282,28 @@ export const OPERATIONS = [
     }
 ];
 
+function convertToNumbers(input) {
```

Comment:
can we have some comments to provide information on what the input is and the expected output

Reply:
done